ddtrace/auto.py

```diff
@@ -0,0 +1 @@
+import ddtrace.bootstrap.sitecustomize  # noqa
```

Comment:
I'm not a big fan of this `auto` approach, but if the rest of the team is on board with it then I'm fine with it too.